bCNC/CNC.py

```diff
@@ -190,7 +190,7 @@ def load(self, filename=None):
 		def read(f):
 			while True:
 				line = f.readline()
-				if len(line)==0: raise
+				if len(line)==0: raise Exception()
```

Comment:
What is this for?

Reply:
The Original implementation was generating a TypeError
  File "C:\projects\bCNC\bCNC\CNC.py", line 193, in read
    if len(line)==0: raise
TypeError: exceptions must be old-style classes or derived from BaseException, not NoneType

Now it generates a normal Exception. An assert would be better. I will change to assert, which reflects the usecase more.
  File "C:\projects\bCNC\bCNC\CNC.py", line 193, in read
    if len(line)==0: raise Exception()
Exception

Reply:
oh, i didn't saw there was `raise` already... makes sense. thanks.